src/app.js

```diff
@@ -239,6 +239,13 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
   const pluginLoader = appManager.pluginLoader
   const workspace = pluginLoader.get()
   const engine = new Engine(appManager)
+  engine.setPluginOption = (profile) => {
+    if (profile.kind === 'provider' ||
```

Comment:
It's a little bit weird to have this kind of indentation ^^.
If you have an automatic corrector you can do : 
```typescript
({ kind, name }) => {
  if (kind === 'provider' || name === 'LearnEth') {
    ...
 }
 ...
}
```

Reply:
ok sure